### PR TITLE
Issue #232. add config to remove email verification check

### PR DESCRIPTION
**Pull request by MufengNiu**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -37,6 +37,11 @@ DB_SCHEMA=public
 # -- Mail settings --
 # -------------------
 
+# Whether to check email verification for new users.
+# 1: check
+# 0: Do not check
+CHECK_VERIFIED=0
```
Comment (yangli0516) [inline .env.example:43]: Please change this name to `NEW_ACCOUNT_EMAIL_VERIFICATION` and use `0` for disable and `1` for enable.

File: config/auth.php [hunk 2]
```diff
@@ -18,6 +18,12 @@
         'passwords' => 'users',
     ],
 
+
+    # Whether to check email verification for new users.
+    # 1: check
+    # 0: Do not check
+    'check_verified' => env('CHECK_VERIFIED', 0),
```
Comment (yangli0516) [inline config/auth.php:25]: Please change the name to `new_account_email_verification`, set default to `0`.

**Review by yangli0516 — CHANGES_REQUESTED** [review]
I think this can be done through the user creation level instead of messing with the middlewares. Inside `\TLCMap\Http\Controllers\Auth\RegisterController::create` you can check the configuration value. If it's enabled, no change at all. If it's disabled, set `is_active` to true when creating the user record.

**Comment by MufengNiu** [comment]
> I think this can be done through the user creation level instead of messing with the middlewares. Inside `\TLCMap\Http\Controllers\Auth\RegisterController::create` you can check the configuration value. If it's enabled, no change at all. If it's disabled, set `is_active` to true when creating the user record.

I think it is checking the value of "email_verified_at ". If we set this value at the time of account creation, wouldn't it remove the entire email verification process? My understanding was that the goal is to grant access to certain features without requiring email verification.

**Comment by yangli0516** [comment]
> > I think this can be done through the user creation level instead of messing with the middlewares. Inside `\TLCMap\Http\Controllers\Auth\RegisterController::create` you can check the configuration value. If it's enabled, no change at all. If it's disabled, set `is_active` to true when creating the user record.
> 
> I think it is checking the value of "email_verified_at ". If we set this value at the time of account creation, wouldn't it remove the entire email verification process? My understanding was that the goal is to grant access to certain features without requiring email verification.

Ok, leave it like this then. This may lead to some side effects as the middleware is not doing its work due to the configuration. And later on it could be misused. I will write some notes under the issue.

**Review by yangli0516 — APPROVED** [review]

**Pull request merged**